examples/elm-square-web-sdk/src/elm-square-web-sdk.js

```diff
@@ -0,0 +1,76 @@
+/* @elm-pkg-js
+port wolfadex_elm_square_web_sdk_to_js : () -> Cmd msg
+ */
+
+/**
+ * We're using https://www.fast.design/docs/fast-element/getting-started to create a basic
+ * custom element wrapper around the Square Web Payments SDK.
+ */
+
+exports.init = async function (app) {
+  // import("https://sandbox.web.squarecdn.com/v1/square.js").then(() => {
+  //   import("hhttps://unpkg.com/@microsoft/fast-element").then((FAST) => {
+  //     initSquareWebSdk(app, FAST);
+  //   });
+  // });
+
+  /**
+   * There are currently technical issues which prevent
+   * using `import("https:...")` in elm-pkg-js. To work
+   * around this you can inject script tags into the
+   * page <head />.
+   */
+  var squareWebSdk = document.createElement("script");
+  squareWebSdk.type = "text/javascript";
+  squareWebSdk.src = "https://sandbox.web.squarecdn.com/v1/square.js";
+  document.head.appendChild(squareWebSdk);
+
+  var fastScript = document.createElement("script");
+  fastScript.type = "module";
+  fastScript.innerHTML = `
+  import * as FAST from "https://unpkg.com/@microsoft/fast-element";
+  window.FAST = FAST;`;
+  document.head.appendChild(fastScript);
+
+  // This delay is to give time for the injected scripts
+  // to download and run. Not ideal, but it does work.
+  setTimeout(() => initSquareWebSdk(app, window.FAST), 1000);
```

Comment:
This is unfortunate 🤔 does the Square SDK not have an onloaded event?

Is it possible to use the JS one instead? I.e.

```
squareWebSdk.onload = function() { initSquareWebSdk(app, window.FAST) };
document.head.appendChild(squareWebSdk);
...
```

Reply:
This never ocurred to me! I'll try it out quick and update accordingly

Reply:
I was able to switch to `onload` for the Square SDK, but not for FAST. Not sure if you have a recommendation for a custom element lib that doesn't use modules? I was having a hard time finding one that would work well for this example. I ask because `onload` doesn't work for the module script because it has to load and then add itself to the global scope, which causes a race condition with actually building the component.

Reply:
> Not sure if you have a recommendation for a custom element lib that doesn't use modules?

No, sorry :( I wasn't aware custom elements needed extra deps, is there a lot of complexity in doing a custom element by hand?

Reply:
It's not terrible by hand, I'll try and do that and see how safe I can make it. It can get kinda lengthy if you properly hand all the connecting and disconnecting. I'll try to have something by the end of the weekend.

Reply:
No rush @wolfadex!

Reply:
@wolfadex just stumbled through this again passing by – was this something you managed to look into any further? :)